world/zonelist.cpp

```diff
@@ -43,7 +43,7 @@ ZSList::ZSList()
 	memset(pLockedZones, 0, sizeof(pLockedZones));
 
 	m_tick = std::make_unique<EQ::Timer>(5000, true, std::bind(&ZSList::OnTick, this, std::placeholders::_1));
-	m_keepalive = std::make_unique<EQ::Timer>(1500, true, std::bind(&ZSList::OnKeepAlive, this, std::placeholders::_1));
+	m_keepalive = std::make_unique<EQ::Timer>(1000, true, std::bind(&ZSList::OnKeepAlive, this, std::placeholders::_1));
```

Comment:
Possible rebase artifact, the master PR was 1500 (just making sure it was intentional or if it matters)

Reply:
They were all 1000ms 

https://github.com/EQEmu/Server/pull/1502/files